launch/dynamo-run/src/subprocess/trtllm_inc.py

```diff
@@ -253,16 +433,46 @@ def cmd_line_args():
     parser.add_argument(
         "--publish-events-and-metrics",
         action="store_true",
-        help="Publish events and metrics to the dynamo components.",
+        help="Publish events and metrics to the dynamo components. Note: This is not supported when running in prefill disaggregation mode.",
+    )
+    parser.add_argument(
+        "--disagg-mode",
```

Comment:
If you call this `--task` we might be able to re-use it for embeddings and other engines types. `--task embedding`, something like that, see https://github.com/ai-dynamo/dynamo/pull/1290

Reply:
Should be resolved by this: [93ce554](https://github.com/ai-dynamo/dynamo/pull/1355/commits/93ce5542100229d87596b8115aaa68983376ed68)